.circleci/config.yml

```diff
@@ -124,17 +131,46 @@ commands:
     steps:
        # Bundler
       - restore_cache:
-          keys: 
+          keys:
             - gem-cache-{{ checksum "<< parameters.directory >>/Gemfile.lock" }}
-      - run: 
+      - run:
           name: Bundle install
           working_directory: << parameters.directory >>
           command: bundle install --clean --path vendor/bundle
       - save_cache:
           key: gem-cache-{{ checksum "<< parameters.directory >>/Gemfile.lock" }}
           paths:
             - vendor/bundle
-  
+
+  set_up_git_user:
+    description: Set git config parameters
+    steps:
+      - run: git config --global user.email "dev+ci@revenuecat.com"
```

Comment:
not needed for this PR, but we should probably move this stuff to env variables

Reply:
Actually, I will probably change it for this PR, since it should be done like that yeah.

Reply:
And this whole command can be removed if using our orb 👍